backend/tests/v5/unit/middleware/dataConverter/outputs/teamspaces/projects/models/commons/modelSettings.test.js

```diff
@@ -15,6 +15,7 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+const { MODEL_TYPES } = require('../../../../../../../../../../src/v5/models/modelSettings.constants');
```

Comment:
use the src variable (i'm losing track how many times I've had to point this out 🙈 )

Reply:
Its auto generated and I keep forgetting to change it 😭